test/test_examples_dg_multi.jl

```diff
@@ -0,0 +1,120 @@
+module TestExamplesDGMulti
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "dg_multi")
+
+@testset "DGMulti" begin
+  @trixi_testset "Euler triangular mesh" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
```

Comment:
Just a recommendation, since I think it makes it easier to associate failed tests with the corresponding elixir without having to look up the corresponding elixir in the test files. This would also apply to the other tests of course.
```suggestion
  @trixi_testset "elixir_euler_weakform_2d.jl" begin
    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
```